### PR TITLE
Add Proccessed file Caching

### PR DESCRIPTION
**Pull request by mssimi**

### CODE REVIEW

File: rector-ci.php [hunk 1]
```diff
@@ -61,4 +61,5 @@
 
     # so Rector code is still PHP 7.2 compatible
     $parameters->set(Option::PHP_VERSION_FEATURES, '7.2');
+    $parameters->set(Option::ENABLE_CACHE, true);
```
Comment (mssimi) [inline rector-ci.php:64]: @TomasVotruba Not sure here, we can turn that on, but dead code is being skipped.

Comment (TomasVotruba) [reply rector-ci.php:64]: Sure we can :) 
I'd make it `false` by default before merging.

How about warning people in command line about the `dead code` being skipped on cache?

Comment (mssimi) [reply rector-ci.php:64]: There already is warning, but it just says number of rectors skipped, do you want to warn with list of rectors being skipped?

Comment (mssimi) [reply rector-ci.php:64]: It is false by default even if I just remove it, should I rather keep it with false or just remove it?

Comment (TomasVotruba) [reply rector-ci.php:64]: Ah, great :)  I confused this with internal config. All good then :+1:

**Comment by TomasVotruba** [comment]
Thank you :+1:

**Pull request merged**